README.md

```diff
@@ -361,6 +361,9 @@ These are the available config options for making requests. Only the `url` is re
   // `maxContentLength` defines the max size of the http response content in bytes allowed
   maxContentLength: 2000,
 
+  // `maxBodyLength` defines the max size of the http request content in bytes allowed
```

Comment:
Is it `Node only`? Glad to see that is mentioned.

Reply:
@gualopezb And ping for here.

Reply:
Not sure what you mean.

Reply:
I mean this option is `Node only`. Better to mention it in the description.

Reply:
Gotcha. PR updated.

Reply:
@gualopezb Now we have 2 lines of comments for `maxBodyLength `. Is one of them duplicated?

Reply:
@chinesedfan you're absolutely right, i just updated it